runtime/browser/android/intercepted_request_data.h

```diff
@@ -1,43 +0,0 @@
-// Copyright (c) 2012 The Chromium Authors. All rights reserved.
```

Comment:
it's no need to keep InterceptedRequestData, for it's part of new structure. 
Xiaosong's suggestion is that use XWalkWebResourceResponse internal and external, so that not use webkit WebResourceResponse which is SDK version related.
Currently, it's was blocked by refection script issue.